src/CitizenProject/CitizenProjectMessageNotifier.php

```diff
@@ -3,18 +3,22 @@
 namespace AppBundle\CitizenProject;
 
 use AppBundle\Committee\CommitteeManager;
+use AppBundle\Coordinator\Filter\AbstractCoordinatorAreaFilter;
 use AppBundle\Entity\Adherent;
 use AppBundle\Entity\CitizenProject;
 use AppBundle\Events;
 use AppBundle\Mailer\MailerService;
 use AppBundle\Mailer\Message\CitizenProjectApprovalConfirmationMessage;
 use AppBundle\Mailer\Message\CitizenProjectCommentMessage;
 use AppBundle\Mailer\Message\CitizenProjectCreationConfirmationMessage;
+use AppBundle\Mailer\Message\CitizenProjectCreationCoordinatorNotificationMessage;
 use AppBundle\Mailer\Message\CitizenProjectCreationNotificationMessage;
 use AppBundle\Mailer\Message\CitizenProjectNewFollowerMessage;
 use AppBundle\Mailer\Message\CitizenProjectRequestCommitteeSupportMessage;
+use AppBundle\Repository\AdherentRepository;
 use OldSound\RabbitMqBundle\RabbitMq\ProducerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
```

Comment:
useless

Reply:
This use is used by : `UrlGeneratorInterface::ABSOLUTE_URL`